distributions/aws/test/e2e/resources/external/secrets_store_csi_driver.py

```diff
@@ -0,0 +1,22 @@
+from e2e.utils.utils import kubectl_apply
+
+RBAC_SECRETS_PROVIDER_CLASS = "https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/v1.0.0/deploy/rbac-secretproviderclass.yaml"
+
+CSI_DRIVER = "https://raw.githubusercontent.com/kubernetes-sigs/secrets-store-csi-driver/v1.0.0/deploy/csidriver.yaml"
```

Comment:
Can you change this variable name to indicate which CSI_DRIVER

Reply:
sure